openaerostruct/aerodynamics/tests/test_geometry.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 
 import openmdao.api as om
-from openmdao.utils.assert_utils import assert_check_partials, assert_rel_error
+from openmdao.utils.assert_utils import assert_check_partials, assert_near_equal
```

Comment:
Why the switch?

Reply:
`assert_rel_error` was deprecated in OpenMDAO 3.1.0 (now on 3.16.0)

Reply:
Ah okay. Their docs should probably state that then, it wasn't obvious to me. What's the difference between the two?

Reply:
It looks like it has something to do with OpenMDAO's internal test case structure. If you wanna look at the source code, [see here](https://github.com/OpenMDAO/OpenMDAO/blob/cc3b5c3b7dca870005cfc1be761746f09a26ff67/openmdao/utils/assert_utils.py). The deprecation is listed [here](https://github.com/OpenMDAO/OpenMDAO/blob/302cd32109d4450de7f9f0a1de04278516cecfcb/release_notes.md) in version 3.1.0. Other than the deprecation warning, this is the only place I've found it.

Reply:
I just diffed the code and they are very similar, so I'm happy with the change.